sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -66,8 +66,21 @@ class Mean(object):
   """Combiners for computing arithmetic means of elements."""
   class Globally(ptransform.PTransform):
     """combiners.Mean.Globally computes the arithmetic mean of the elements."""
+    def __init__(self, has_defaults=True, *args, **kwargs):
```

Comment:
Can you add Pydoc for this? And I don't think we need `args` and `kwargs` to be added. Probably take only has_defaults.

Reply:
I took out the `args` and `kwargs`, but didn't add the Pydoc to keep it consisten [`CombineGlobally`](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/transforms/core.py#L1834) (doesn't have it either). Let me know if you want me to add them.

Thanks again